### PR TITLE
[MAJOR] STAGING -> MAIN

### PR DESCRIPTION
**Pull request by JayeshVP24**

### CODE REVIEW

**Review by demonkillerr — CHANGES_REQUESTED** [review]
Kindly make the following changes

1. Following changes are global

    - [x] Route the links correctly to training, internship and [open source](https://opensource.tcetmumbai.in/) pages
    - [x] Fix spelling mistakes through out the website
    - [x] Add primary navbar links
    - [x] Add Favicon
    - [x] Add About to PR title

2. Training Page

    - [x] Replace Hacknova with Solving for India GFG hackathon
    - [x] Replace CC with GFG event

3. About Page

    - [x] Submit button should work (mailto)

4. Internship Page

    - [x] Fix no of students for larger screens

**Comment by riyavishwa981** [comment]
* Landing Page

1. make some space between view more and the image below it 
image

* Training Page - Perfect
* About Tnp Page - Perfect
* Internship Page - Perfect

**Comment by demonkillerr** [comment]
Res: 1920x1080

image

**Comment by demonkillerr** [comment]
Add `target=_blank` on all external links

**Review by riyavishwa981 — APPROVED** [review]

File: src/pages/index.astro [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+import Testimonial from "../components/Testimonials/index.astro";
+import HeroSection from "../components/Hero/HeroSection.astro";
+import TopPlaceStudents from "../components/TopPlacedStudents/index.astro";
+import StudentAward from "../components/StudentAward/StudentAward.jsx";
 ---
-
 <Layout
   title="TCET - Training and Placement Cell"
-  description="The official Website of Training and Placement Cell of Thakur College of Engineering and Technology"
+  description="The official Website of Training and Placement Cell of the amazing Thakur College of Engineering and Technology"
```
Comment (demonkillerr) [inline src/pages/index.astro]: need to revert this

**Review by demonkillerr — CHANGES_REQUESTED** [review]

**Review by demonkillerr — CHANGES_REQUESTED** [review]
Important Change:

- [ ] Replace all dummy text with actual text
- [x] Replace all dummy images with real images

**Comment by JayeshVP24** [comment]
Ready to merge

**Review by vish-rt — APPROVED** [review]
Looks good to me!

**Review by demonkillerr — APPROVED** [review]

**Review by demonkillerr — APPROVED** [review]

**Comment by demonkillerr** [comment]
Final Production Push for TNP Website. End of beta-testing phase.

Website can be accessed at [https://tnp.tcetmumbai.in/](https://tnp.tcetmumbai.in/).

**Pull request merged**